src/wallet/wallet.cpp

```diff
@@ -2320,8 +2326,16 @@ bool CWalletTx::IsTrusted(interfaces::Chain::Lock& locked_chain) const
         if (parent == nullptr)
             return false;
         const CTxOut& parentOut = parent->tx->vout[txin.prevout.n];
+        // Check that this specific input being spent is trusted
         if (pwallet->IsMine(parentOut) != ISMINE_SPENDABLE)
             return false;
+        // If we've already trusted this parent, continue
+        if (trustedParents.count(parent->GetHash()))
+            continue;
```

Comment:
nit, join with line above - to follow correct format.

Reply:
Not sure what you mean

Reply:
got it; went ahead and did this in the entire function (rather than leave it half-styled)

Reply:
Yup, that was the idea.